packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -765,6 +781,54 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     ev.preventDefault();
   }
 
+  @autobind
+  private _onTypeKeyDown(ev: React.KeyboardEvent<HTMLDivElement>) {
+    // Ignore non-displayed, modifier, and special keys
+    if (!/^[a-zA-Z0-9-_ ]$/.test(ev.key)) {
```

Comment:
I'm not sure that this will work for other languages.

Reply:
Also I'm not sure that this will catch something "Enter" which is what the key will be. You could try to filter them out based on KeyCodes.